packages/webview_flutter/webview_flutter/README.md

```diff
@@ -45,54 +77,141 @@ android {
 }
 ```
 
-### Using Virtual displays
+### Platform Specific Features
 
-1. Set the correct `minSdkVersion` in `android/app/build.gradle` (if it was previously lower than 20):
+Many classes have a subclass or an underlying implementation that provides access to platform
+specific features.
 
-    ```groovy
-    android {
-        defaultConfig {
-            minSdkVersion 20
-        }
-    }
-    ```
-
-2. Set `WebView.platform = AndroidWebView();` in `initState()`.
-    For example:
+To access platform specific features, start by including the import for the desired platform:
 
-    ```dart
-    import 'dart:io';
-
-    import 'package:webview_flutter/webview_flutter.dart';
+<?code-excerpt "main.dart (platform_imports)"?>
+```dart
+// Import for Android features.
+import 'package:webview_flutter/android.dart';
+// ···
+// Import for iOS features.
+import 'package:webview_flutter/wkwebview.dart';
```

Comment:
We should remove these files entirely, and show importing from the packages instead, because if we re-export platform-specific packages, then any breaking change to those packages' platform-specific APIs will require a breaking change to the app-facing package as well, which we don't want.

Using the implementation packages directly for platform features is the pattern we're using in other plugins (local_auth, IAP, etc.)

Reply:
I agree with this approach. I like the app-facing package to remain mostly unaware of its implementations.

But to do this, I also had to add `LoadRequestParams` and `JavaScriptChannelParams` to the `webview_flutter.dart` export. I'm considering some other options,

1. We could export `LoadRequestParams` and `JavaScriptChannelParams` in the platform implementations instead.
2. Create a `import 'package:webview_flutter/platform_interface.dart'` export file.
3. Require apps to import the platform interface when importing the platform implementations.
4. Leave it as is. 

My preference is to export these classes in the platform implementation classes. This would allow us import a minimum needed in the app-facing package and not require any additional work for a user to use the platform implementation classes.

Reply:
Agreed, 1 seems like the best option. It also seems like the most consistent with the general policy that an interface should export all the types used in its public API (comparable to the lint about not using private classes in public APIs).